Re-enable relationship counts from count store

systay opened the pull request.

Commit:
Make sure to inform the monitor when compiled runtimes are used

Commit:
Enable relationship count from count store in the compiled runtime

jexp commented:
Nice!

Von meinem iPhone gesendet

> Am 15.02.2017 um 15:22 schrieb Andres Taylor <notifications@github.com>:
> 
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/neo4j/neo4j/pull/8829
> 
> Commit Summary
> 
> Make sure to inform the monitor when compiled runtimes are used
> Enable relationship count from count store in the compiled runtime
> File Changes
> 
> M community/cypher/cypher/src/test/scala/org/neo4j/cypher/LdbcQueries.scala (4)
> M enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/LdbcAcceptanceTest.scala (7)
> M enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/LoadCsvAcceptanceTest.scala (2)
> M enterprise/cypher/cypher-compiled-runtime-3.2/src/main/scala/org/neo4j/cypher/internal/compiled_runtime/v3_2/BuildCompiledExecutionPlan.scala (8)
> M enterprise/cypher/cypher-compiled-runtime-3.2/src/main/scala/org/neo4j/cypher/internal/compiled_runtime/v3_2/codegen/LogicalPlanConverter.scala (2)
> M enterprise/cypher/cypher-compiled-runtime-3.2/src/main/scala/org/neo4j/cypher/internal/compiled_runtime/v3_2/codegen/ir/RelationshipCountFromCountStoreInstruction.scala (3)
> M enterprise/cypher/cypher-compiled-runtime-3.2/src/main/scala/org/neo4j/cypher/internal/compiled_runtime/v3_2/codegen/ir/expressions/Literal.scala (13)
> A enterprise/cypher/cypher-compiled-runtime-3.2/src/test/scala/org/neo4j/cypher/internal/compiled_runtime/v3_2/CompiledRuntimeContextHelper.scala (56)
> M enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v3_2/codegen/GeneratedMethodStructure.scala (9)
> M enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v3_2/codegen/GeneratedQueryStructure.scala (7)
> M enterprise/cypher/cypher/src/test/scala/org/neo4j/cypher/MutatingIntegrationTest.scala (2)
> A enterprise/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiled_runtime/v3_2/BuildCompiledExecutionPlanTest.scala (84)
> Patch Links:
> 
> https://github.com/neo4j/neo4j/pull/8829.patch
> https://github.com/neo4j/neo4j/pull/8829.diff
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

henriknyman reviewed (commented):
Looks good. We just need to change some tests to run with all runtimes.

davidegrohmann committed:
Fix tests after improving NewPlannerTestSupport

After having add a check for checking with runtime was used to runt he
given query many tests started failing
